src/main/java/com/elastic/support/diagnostics/commands/RunKibanaQueries.java

```diff
@@ -0,0 +1,256 @@
+package com.elastic.support.diagnostics.commands;
+
+import com.elastic.support.Constants;
+import com.elastic.support.diagnostics.DiagnosticException;
+import com.elastic.support.diagnostics.JavaPlatform;
+import com.elastic.support.diagnostics.ProcessProfile;
+import com.elastic.support.diagnostics.chain.DiagnosticContext;
+import com.elastic.support.rest.RestClient;
+import com.elastic.support.rest.RestEntry;
+import com.elastic.support.rest.RestResult;
+import com.elastic.support.rest.RestEntryConfig;
+import com.elastic.support.util.*;
+import com.fasterxml.jackson.databind.JsonNode;
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.Iterator;
+
+import java.io.FileWriter;
+import java.io.IOException;
+
+/**
+ * RunKibanaQueries executes the version-dependent REST API calls against Kibana.
+ */
+public class RunKibanaQueries extends BaseQuery {
+
+    private static final Logger logger = LogManager.getLogger(RunKibanaQueries.class);
+
+    /**
+    * Create a new ProcessProfile object and extract the information from fileName to get PID and OS.
```

Comment:
Line these up consistently please (don't want to just keep filling up the review).